docs/L1_L2_Messages.md

```diff
@@ -0,0 +1,245 @@
+---
+id: L1_L2_Messages
+title: Messaging Between Layers
+sidebar_label: Messaging Between Layers
+---
+
+## Standard Arbitrum Transactions: Calls from clients
+
+Standard, client-generated transaction calls on the Arbitrum chain are sent through the EthBridge using Inbox.sendL2Message:
+
+```solidity
+function sendL2Message(address chain, bytes calldata messageData) external;
+```
+
+Generally calls will come in batches from an aggregator as described in [Transaction Lifecycle](Tx_Lifecycle.md).
+
+However, the Arbitrum protocol also offers ways passing messages between the layer 1 and layer 2 chains.
+
+The most common use-case for direct inter-chain communication is depositing and withdrawing assets; this, however, is only one specific application of generalized cross-chain contract calls that Arbitrum supports. This page covers the generalized protocol; for further explanation, see [Inside Arbitrum: Bridging](https://developer.offchainlabs.com/docs/inside_arbitrum#bridging).
```

Comment:
Can we make this a relative link, ie `[Inside Arbitrum: Bridging](Inside_Arbitrum.md#bridging)`